source/includes/stackpath/_delivery_rules.md

```diff
@@ -49,6 +49,86 @@ Attributes | &nbsp;
 `stackId`<br/>*string* | The ID of the stack that the delivery rule belongs to.
 `siteId`<br/>*string* | The ID of the site that the delivery rule is applied to.
 
+<!-------------------- CREATE A DELIVERY RULE -------------------->
+
+### Create a delivery rule
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/deliveryrules?siteId=f9dea588-d7ab-4f42-b6e6-4b85f273f3db"
+```
+
+> Request body example for creating a delivery rule:
+
+```js
+{
+ "name": "rule",
+  "deliveryRuleConditions": [
```

Comment:
I just made a small change for it to be `conditions` and `actions` instead of `deliveryRuleConditions` and `devlieryRuleActions` felt the delivery rule is a bit redundant.

Reply:
@jasondias9 Fixed in 94b094b2ee22e987e466f29352a7cd72dfa72d20